pkg/planner/core/fragment.go

```diff
@@ -52,7 +53,7 @@ type Fragment struct {
 	CTEReaders        []*PhysicalCTE              // The receivers for CTE storage/producer.
 
 	// following fields are filled after scheduling.
-	ExchangeSender *PhysicalExchangeSender // data exporter
+	Sink MPPSink // data exporter
```

Comment:
This name is to be compatible with tiflash's concept `Sink` and `Source`.
Other names are welcome.